recipes/nspr/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+cd nspr
+
+export ARCH_FLAG=""
+if [[ "${ARCH}" == 64 ]]
+then
+    export ARCH_FLAG="--enable-64bit"
+fi
+
+./configure --prefix=${CONDA_PREFIX} $ARCH_FLAG
```

Comment:
We can just use `PREFIX` for simplicity (`conda-build` guaranteed). Also would quote this (in case someone builds locally and has spaces in their path).

Reply:
Changed! Ty